environment

```diff
@@ -31,6 +31,11 @@ FUS_TERRAFORM_BACKEND_BUCKET_TEMPLATE=terraform-tsmith
 FUS_ADMIN_EMAILS=test_email@domain.com
 DEBUG=0
 AWS_SDK_LOAD_CONFIG=1 # Needed for Terraform to correctly use AWS assumed roles
+
+# tags for AWS resources
+FUS_PROJECT_TAG=dcp
+FUS_OWNER_TAG=user@email.com
```

Comment:
this needs to be the fusillade contact email

Reply:
It's replaced with my email when it's deployed. Is that ok?

Reply:
👍